test/ase_pycall.jl

```diff
@@ -0,0 +1,24 @@
+@testset "ASE" begin
+    include("test_utils.jl")
+    using PyCall
+
+    ase = pyimport("ase")
+
+    h2 = ase.Atoms("H2", [(0, 0, 0), (0, 0, 0.74)])
+    h2.center(vacuum=2.5)
+
+    @testset "EMT" begin
+        emt = pyimport("ase.calculators.emt")
+        h2.calc = emt.EMT()
```

Comment:
just a note: it used to be that calculators require setting with atoms.set_calculator(calc). EMT might be a rare exception where you can directly set it like this